src/App.tsx

```diff
@@ -1,9 +1,25 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
+function lowerCompare(state1: string, state2: string): boolean {
+  return state1.toLowerCase().includes(state2.toLowerCase());
+}
+
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+
+  const visibleMovies = [...moviesFromServer]
```

Comment:
filter is a non-mutable method so you don't need a spread here. (checklist point 3)

Reply:
Agreed.